tests/tests_basics_files.yml

```diff
@@ -72,6 +72,10 @@
       command: /bin/logger -i -p local6.info -t testTag0 testMessage0000
       changed_when: false
 
+    - name: Pause 1 sec before checking "{{ __default_system_log }}"
+      pause:
+        seconds: 1
+
     - name: Check the test log message in {{ __default_system_log }}
       command: /bin/grep testMessage0000 {{ __default_system_log }}
       changed_when: false
```

Comment:
I think it would be better to use `until` on the Check task - https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#retrying-a-task-until-a-condition-is-met - that is - the check task would retry N times with a M second pause between each retry

Reply:
Thanks a lot, @richm. Updated the commit.

Reply:
Hi @xjezda00, ^^^^^